src/lib/es2022.sharedmemory.d.ts

```diff
@@ -0,0 +1,7 @@
+interface Atomics {
+    /**
+     * A non-blocking, asynchronous version of wait and usable on the main thread.

```

Comment:
```suggestion
     * A non-blocking, asynchronous version of wait, usable on the main thread.
```

or maybe ".. version of wait which is usable ..."

Reply:
Awesome! ``".. version of wait which is usable ..."``sounds really good :)